java/delivery-api/GetItem.java

```diff
@@ -2,10 +2,14 @@
 // Tip: Find more about Java SDK at https://docs.kontent.ai/java
 import kentico.kontent.delivery.*;
 
+// Initializes a DeliveryClient
 DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");
 
-List<NameValuePair> params = DeliveryParameterBuilder.params().build();
+// Registers the model class for articles
+// Tip: Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
+client.registerType(Article.class);
 
-// Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
-CompletionStage<ArticleItem> item = client.getItem("my_article", ArticleItem.class, params);
+// Gets an article
+CompletionStage<Article> article = client.getItem("my_article", Article.class);
+// To use the code for Android projects, see http://docs.kontent.ai/javarx
```

Comment:
I would discuss where the link leads. Because it looks like - "if you want to use Android , you should use java RX", but Android could be implemented with any library compatible with Java CompletionsStage i.e. in Kotlin using Kotlin coroutines.

I can add the "Android" section to the https://github.com/Kentico/kontent-java-packages/tree/master/kontent-delivery saying something like:

> # Android development
> 
> If you want to use this SDK for Android development, you should:
> * _Use the content from_ https://github.com/Kentico/kontent-java-packages/tree/master/sample-app-android-kotlin#strongly-typed-models-with-models
> * _Use the content from_ https://github.com/Kentico/kontent-java-packages/tree/master/sample-app-android-kotlin#strongly-typed-models-with-models
> 
> For Android development you can use approach compatible  with CompletionStages returned from the Delivery CLient. Most common is Java RX for Java development for ANdroid and Android Coroutines for

and the link to this section.

+ maybe change reference address of the link like i.ee:  `http://docs.kontent.ai/android`

Reply:
A separate section for Android would be most splendid. I can proofread it a bit after you add something. I'll change the link to /android instead of /javarx. :)

Reply:
Section prepared https://github.com/Kentico/kontent-java-packages/pull/123

The link should lead to `https://github.com/Kentico/kontent-java-packages/kontent-delivery#Android-development` once the pull request is published.

Reply:
I've reviewed the new section and updated the code samples. I'd like to resolve this thread and merge the PR.

Reply:
http://docs.kontent.ai/android  should lead to https://github.com/Kentico/kontent-java-packages/tree/master/kontent-delivery#android-development right?